lists/id.csv

```diff
@@ -159,3 +159,20 @@ http://www.zimbio.com/Free+proxy+sites/articles/130/Free+youtube+Proxy,ANON,Anon
 http://xssnews.ru/index.php,GRP,Social Networking,2014-04-15,citizenlab,
 http://zheg.nastie.co.uk,GAME,Gaming,2014-04-15,citizenlab,
 http://zixzaxteam.net,GRP,Social Networking,2014-04-15,citizenlab,
+http://suarapapua.com/,HUMR,Human Rights Issues,2017-04-24,sinarproject,Papua conflict
+http://infopapua.org,HUMR,Human Rights Issues,2017-04-24,sinarproject,Papua conflict
+http://tabloid-wani.com,HUMR,Human Rights Issues,2017-04-24,sinarproject,Papua conflict
+http://papuapost.com,HUMR,Human Rights Issues,2017-04-24,sinarproject,Papua conflict
+http://freepapua.com,HUMR,Human Rights Issues,2017-04-24,sinarproject,Papua conflict
+http://ampnews.org,HUMR,Human Rights Issues,2017-04-24,sinarproject,Papua conflict
+http://www.papuansbehindbars.org/,HUMR,Human Rights Issues,2017-04-24,sinarproject,Papua conflict
+http://tabloidjubi.com/,HUMR,Human Rights Issues,2017-04-24,sinarproject,Papua conflict
+http://www.papua.us/,HUMR,Human Rights Issues,2017-04-24,sinarproject,Papua conflict
+voa-islam.com,REL,Religion,2017-04-24,sinarproject,"Radical Islam with elements of hate speech: https://inet.detik.com/law-amp-policy/d-3386201/setelah-blokir-habib-rizieq-kominfo-kini-tutup-akses-islampos-dkk"
+nahimunkar.com,REL,Religion,2017-04-24,sinarproject,"Radical Islam with elements of hate speech"
+kiblat.net,REL,Religion,2017-04-24,sinarproject,"Radical Islam with elements of hate speech"
+bisyarah.com,REL,Religion,2017-04-24,sinarproject,"Radical Islam with elements of hate speech"
+dakwahtangerang.com,REL,Religion,2017-04-24,sinarproject,"Radical Islam with elements of hate speech"
+islampos.com,REL,Religion,2017-04-24,sinarproject,"Radical Islam with elements of hate speech"
+gensyiah.com,REL,Religion,2017-04-24,sinarproject,"Radical Islam with elements of hate speech"
+abuzubair.net,REL,Religion,2017-04-24,sinarproject,"Radical Islam with elements of hate speech"

```

Comment:
These URLs are missing the `http(s?)://` prefix:

see relevant travis-ci lines:

```
lists/id.csv (line 171): Invalid URL "voa-islam.com"
lists/id.csv (line 172): Invalid URL "nahimunkar.com"
lists/id.csv (line 173): Invalid URL "kiblat.net"
lists/id.csv (line 174): Invalid URL "bisyarah.com"
lists/id.csv (line 175): Invalid URL "dakwahtangerang.com"
lists/id.csv (line 176): Invalid URL "islampos.com"
lists/id.csv (line 177): Invalid URL "gensyiah.com"
lists/id.csv (line 178): Invalid URL "abuzubair.net"
```